### DISCUSSION TITLE
Error: autorun.memo must used in autorun function body

### DISCUSSION THREAD

**Discussion by marspal** [alibaba/formily#2326] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/41223123/137855785-bf5500d8-9160-4cb5-a14d-b4376a79243e.png)

![image](https://user-images.githubusercontent.com/41223123/137855864-beeb15c8-ce0b-4624-a269-d2ffb72151f5.png)

给目标设置默认值 副作用报错

**Comment by zlyyyy** [comment]
老哥，你这问题咋解决的

**Reply by shengbenqiang** [reply]
请求有解决方法嘛？

**Comment by shengbenqiang** [comment]
我也遇到相同的问题了，求解，翻了官网应该没有陪错
